modules/living_spaces_group/living_spaces_group.module

```diff
@@ -1182,12 +1182,18 @@ function living_spaces_group_group_update(GroupInterface $group) {
  */
 function living_spaces_group_query_living_spaces_menus_user_spaces_alter(Drupal\Core\Database\Query\AlterableInterface $query) {
   $query->leftJoin('user__joined_spaces', 'user__joined_spaces', 'user__joined_spaces.joined_spaces_target_id = groups_field_data.id');
-  $query->condition('user__joined_spaces.entity_id', \Drupal::currentUser()->id(), '=');
+
+  $orCondition = $query->orConditionGroup()
```

Comment:
You cannot use camel cases for variables in hooks.

Reply:
Done!